client/src/components/ReviewsFromUS.jsx

```diff
@@ -1,15 +1,254 @@
 import React from 'react';
+import StarRating from 'react-star-rating-component';
+import styled from 'styled-components';
 
-const ReviewsFromUs = (props) => {
-  var reviews = props.reviews;
+// const ReviewsFromUs = (props) => {
+class ReviewsFromUs extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      descReadMoreToggleOpen: false
 
-  return (
-    <div >
-      <div><h3>Top reviews from the United States</h3></div>
+    };
+    this.toggleDescPanel = this.toggleDescPanel.bind(this);
+  };
+  //var reviews = props.reviews[0].productId ? props.reviews[0].productId : 'none';
+  toggleDescPanel(event) {
+    this.setState({
+      descReadMoreToggleOpen: !this.state.descReadMoreToggleOpen
 
-    </div>
+    });
+
+  }
+
+  render() {
+    // console.log('Reviews From US:', props.reviews);
+    // console.log('Reviews From US -1:', props.reviews[0]);
+    const reviews = this.props.reviews ? this.props.reviews : [];
```

Comment:
Same here with defining styled components inside the render method

Reply:
Sure.  I will look into for next PR.